kategory/src/test/kotlin/kategory/data/EvalTest.kt

```diff
@@ -8,7 +8,10 @@ import org.junit.runner.RunWith
 class EvalTest : UnitSpec() {
     init {
 
-        testLaws(MonadLaws.laws(Eval, EvalEq()))
+        testLaws(MonadLaws.laws(Eval, object : Eq<HK<Eval.F, Int>> {
```

Comment:
The Eq instance for Eval should be valid even for the main library. Was this a merge conflict, or is there something wrong with the previous one?